signer/src/network/libp2p/network.rs

```diff
@@ -505,4 +527,184 @@ mod tests {
         gossiped_messages.sort_by_cached_key(|x| x.inner.bitcoin_chain_tip);
         assert_eq!(gossiped_messages, signed_messages);
     }
+
+    #[tokio::test]
```

Comment:
nit: missing the `test` surrounding here? I'm still not sure if we need it or not

Reply:
I usually only add it if/when I need logging, and then I leave it 😅